daemon/cmd/daemon_main.go

```diff
@@ -1370,6 +1373,24 @@ func initEnv(cmd *cobra.Command) {
 			option.EgressMultiHomeIPRuleCompat,
 		)
 	}
+
+	if option.Config.InstallNoConntrackIptRules {
+		// InstallNoConntrackIptRules can only be enabled in direct
+		// routing mode as in tunneling mode the encapsulated traffic is
+		// already skipping netfilter conntrack.
+		if option.Config.Tunnel != option.TunnelDisabled {
+			option.Config.InstallNoConntrackIptRules = false
+			log.Warnf("%s requires the agent to run in direct routing mode. Disabling it.", option.InstallNoConntrackIptRules)
```

Comment:
Given `InstallNoConntrackIptRules` now defaults to false, the warnings need to be fatals. The user explicitly asked for something and we refused to enable, so we should be loud about it.